modules/search/instant-search/components/search-results.jsx

```diff
@@ -4,39 +4,27 @@
  * External dependencies
  */
 import { __, _n, sprintf } from '@wordpress/i18n';
-import { h, Component } from 'preact';
+import { h, Component, Fragment } from 'preact';
 
 /**
  * Internal dependencies
  */
-import SearchResultMinimal from './search-result-minimal';
-import { hasFilter } from '../lib/query-string';
+import SearchResult from './search-result';
 import ScrollButton from './scroll-button';
+import SearchForm from './search-form';
+import SearchSidebar from './search-sidebar';
+import Notice from './notice';
 
 class SearchResults extends Component {
-	renderResult = ( result, index ) => {
-		switch ( this.props.resultFormat ) {
-			case 'engagement':
-			case 'product':
-			case 'minimal':
-			default:
-				return (
-					<SearchResultMinimal
-						index={ index }
-						locale={ this.props.locale }
-						query={ this.props.query }
-						result={ result }
-					/>
-				);
-		}
-	};
-
 	getSearchTitle() {
 		const { total = 0, corrected_query = false } = this.props.response;
 		const hasQuery = this.props.query !== '';
 		const hasCorrectedQuery = corrected_query !== false;
 		const num = new Intl.NumberFormat().format( total );
 
+		if ( total === 0 || this.props.hasError ) {
+			return sprintf( __( 'No results for "%s".', 'jetpack' ), this.props.query );
```

Comment:
is props.query escaped properly?

Reply:
React/Preact will [escape any values embedded in JSX](https://reactjs.org/docs/introducing-jsx.html#jsx-prevents-injection-attacks) before rendering them.